portfolio/src/main/webapp/style.css

```diff
@@ -1,13 +1,13 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 500px;
```

Comment:
No need to it this time. But it will good to have another commit for the style change in this file. The reason is that we can put more information in the commit message, e.g. why we want to change the styling - and it's really related to the new paragraph we add here.